src/app/products/manage-delinquency-buckets/delinquency-bucket/edit-bucket/edit-bucket.component.ts

```
@@ -176,24 +219,43 @@ export class EditBucketComponent implements OnInit {
         ranges: ranges
       };
+    } else if (this.isWorkingCapitalBucket) {
+      const payload = this.bucketForm.getRawValue();
+      const bucketName = payload['name'];
+      return {
+        bucketType: bucketType,
+        name: bucketName,
+        minimumPaymentPeriodAndRule: payload
+      };
+    }
+  }
```

Biome (2.4.7)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)